content/ytt/docs/latest/lang-ref-ytt-schema.md

```diff
@@ -0,0 +1,226 @@
+---
+title: Schema module
+---
+
+## Overview
+
+`ytt` schemas are currently in the **experimental** phase. To use schema features, include `--enable-experiment-schema`.
+
+`ytt`'s Schema feature provides a way to verify data value YAML structures and values with the help of annotations.
+
+A schema allows you to provide default data values, and verify that each data value file provided is type checked and validated against the schema and guaranteed to be proper.
```

Comment:
critique: compound sentence. I think it would help to enumerate, chronologically, how schema participate.